packages/node-runtime-worker-thread/src/index.spec.ts

```diff
@@ -12,10 +12,50 @@ describe('WorkerRuntime', () => {
       const runtime = new WorkerRuntime('mongodb://nodb/', {}, { nodb: true });
       const result = await runtime.evaluate('1+1');
 
-      expect(result.rawValue).to.equal(2);
+      expect(result.printable).to.equal(2);
 
       await runtime.terminate();
     });
+
+    describe('errors', () => {
+      let runtime: WorkerRuntime;
+
+      beforeEach(() => {
+        runtime = new WorkerRuntime('mongodb://nodb/', {}, { nodb: true });
```

Comment:
oh this is soo nice!

Reply:
Not sure what the second argument is though

Reply:
This constructor matches `CliServiceProvider` constructor, so it's `uri`, then `driverOptions`, then `cliOptions`, second being the `driverOptions`, that we don't need for this test

Reply:
Can we get in a situation where the 2 competes? Like `new WorkerRuntime('...', {ssl: true}, {ssl: false})`?

Is the `no-db` a real world use case or is just for testing? maybe we can just start a local connection or figure another way that is not a public option.

Reply:
I don't think so, at least the way they are introduced in `cli-service-provider` would not allow for that I think. This nodb is literally same thing that activates `--nodb` in `mongosh` as the same service provider is used in this runtime. It's definitely not something that can be used in Compass (at least at the moment, unless we want to start showing shell before connecting to the database, which we can now 😉) in any way, but also simplifies writing tests quite a lot